### DISCUSSION TITLE
gradient structure

### DISCUSSION THREAD

**Discussion by yiyuxingchen** [neperfepx/neper#452] [Q&A] [answered]
Hello,everyone!
I want to make such a structure, the upper layer is equiaxed grains, and the lower layer is equiaxed grains plus fibrous structure.I try to generate multiscale tessellations, but the interface between 1-scale grains is not a normal grain boundary, and I don't know how to generate a structure with a mixture of equiaxed grains and fibrous structures.
Thanks in advance.
![微信图片_20220425152907](https://user-images.githubusercontent.com/103731227/165041023-a0a37502-b00c-4730-9dfc-d84ac614fd9b.jpg)
$ neper -T -n 3::1000 -id 1::1 -morphooptiini "coo:file(coo1),weight:0" -morpho "gg::msfile(morpho_msfile3)" -o 3 -statseed x,y,z,w
$ neper -V 3.tess -datacellcol ori -cameraangle 12 -imagesize 600:600 -print img3
coo1:
0.50 0.50 0.15
0.50 0.50 0.50
0.50 0.50 0.85
morpho_msfile3:
1 diameq:lognormal(2,0.35),1-sphericity:lognormal(0.145,0.03)
2 diameq:lognormal(1,0.35),1-sphericity:lognormal(0.145,0.03),aspratio(2,2,0.5)
3 diameq:lognormal(1,0.35),1-sphericity:lognormal(0.145,0.03)

![image](https://user-images.githubusercontent.com/103731227/165041195-58a7c989-5503-4988-a606-472459dcde26.png)

**Comment by fzw9381** [comment, marked as answer]
It's interesting. But I don't believe the current version of Neper  can do this.

**Reply by hongtaoliuxing** [reply]
Do you solve the problem of how to build gradient crystal model with neper？
